### PR TITLE
feat: Raw data pointer support for Ceres

### LINKED ISSUE #733: Raw data pointer support for Ceres

### PR DESCRIPTION
**Pull request by janmichek**
## Description
resolves #733

## Demo
image

https://pr-761-aescan-testnet.stg.service.aepps.com/names/testaensrawpointers.chain
https://pr-761-aescan.stg.service.aepps.com/names/c.chain

## Checklist:

- [x] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)

### CODE REVIEW

**Review by Liubov-crypto — CHANGES_REQUESTED** [review]
I got some issue when tried to check info about name that is in auction: 


https://github.com/aeternity/aescan/assets/69896204/afca465a-9ca7-4083-97ca-ef3d04657135



au1

@janmichek  please check it

**Comment by janmichek** [comment]
> I got some issue when tried to check info about name that is in auction:
> 2024-05-14.5.01.35.mov
> 
> au1
> 
> @janmichek please check it

Good catch, it's fixed now

File: src/utils/adapters.js [hunk 1]
```diff
@@ -297,7 +308,7 @@ export function adaptName(name, blockHeight, blockTime) {
     },
     customPointers,
   }
-
+  console.log('formattedName', formattedName)
```
Comment (michele-franchi) [inline src/utils/adapters.js:311]:
```suggestion
```

File: src/utils/adapters.js [hunk 2]
```diff
@@ -306,7 +317,7 @@ export function adaptName(name, blockHeight, blockTime) {
     })
     formattedName.activatedHeight = name.info.activeFrom
   }
-
+  console.log('formattedName', formattedName)
```
Comment (michele-franchi) [inline src/utils/adapters.js:320]:
```suggestion
```

**Review by michele-franchi — APPROVED** [review]
Looks good, just need to remove some console.log @janmichek.

File: src/utils/adapters.js (hunk 2 above)
Comment (janmichek) [reply src/utils/adapters.js]: removed

File: src/utils/adapters.js (hunk 1 above)
Comment (janmichek) [reply src/utils/adapters.js]: removed

**Comment by janmichek** [comment]
@michele-franchi fixed, can you check again, please?

**Review by michele-franchi — APPROVED** [review]
LGTM

**Pull request merged**